### DISCUSSION TITLE
JLyrics dependency error

### DISCUSSION THREAD

**Discussion by pratyushgta** [jagrosh/MusicBot#1065] [Troubleshooting]
![2021-12-06 (2)](https://user-images.githubusercontent.com/66841352/144763336-4226c094-459a-480a-b1d8-08364a83b133.png)

I'm creating a lyrics discord bot but while adding the dependencies for JLyrics API, I'm getting the following error related to the SNAPSHOT version. Any workarounds for this?

**Comment by jagrosh** [comment]
This doesn't seem to be related to JMusicBot, as it is about a different project, and building JMusicBot from source is not supported.

**Reply by pratyushgta** [reply]
I understand that. But I did not find a discussions forum related to JLyrics.

**Reply by YJSoft** [reply]
`building JMusicBot from source is not supported.`
In other words, there are no official way to compile source code. You have to figure out by yourself.

**Reply by pratyushgta** [reply]
Alright. Thanks.
